stylesheets/card/card.css

```diff
@@ -133,8 +132,23 @@ button
 }
 .horizontal-card
 {
-    max-width:35rem;
+    display:flex;
+    flex-direction: row;
+    max-width:50rem;
     height:auto;
+    gap:1.3rem;
+    box-shadow: 2px 2px 4px var(--light-stone);
+}
+.horizontal-card .card-image
+{
+    height:100%;
+    min-width:14rem;
+}
+.button-container
+{
+    display: flex;
+    flex-direction: row;
```

Comment:
no need to write default flex direction + as root is set 62.5 box-size no need of explicit 1rem